testing/integration_test.go

```diff
@@ -87,7 +94,7 @@ mkdir -p "$TESTDIR"
 cp "testing/busybench.js" "$TESTDIR"
 cd "$TESTDIR"
 
-npm install "$PROFILER" >/dev/null
+retry npm install "$PROFILER" >/dev/null
```

Comment:
This installs local archive so shouldn't have any network interactions? Or it has for deps?

Reply:
I think this is local, but we've also had a flake here. 
I think it makes sense to retry this and npm pack together.

Reply:
We've had a flake with `npm install "$PROFILER"` (continuous build 899). But when looking at that flake, I don't know that can tell whether is occurred because npm pack did something bad and so we tried to install a malformed file, or if it occurred because something went wrong with npm install.

Reply:
Looked more into the flake from continuous build 899 -- I think that the problem was in the `npm install` step, but that retrying likely would not have helped. Generally calling npm install will initialize a package.json if there isn't one. I'm guessing there may have been a problem with npm install which changed this behavior during this test. 

Done.